contents/javascript/Javascript-framework-VueJs.md

```diff
@@ -78,13 +78,15 @@ In this case, only the `root` component can be accessed in VueJs while the rest
 <b>Step 4:</b> Pass the message to the HTML file using double curly brackets.
 ```HTML
   <div id="root">
-    <h2>{{ message }}</h2>
+    <h2>{ message }</h2>
```

Comment:
Is this correct? I thought Vue uses two braces?

Reply:
Yes, the correct syntax is `{{ message }}`, however Markbind doesn't allow this to be rendered, causing this whole part to disappear altogether.

Reply:
Perhaps use `{\{` or something like that and mention that in the actual code it should be replaced with `{{`?